.pfnci/test.sh

```diff
@@ -5,12 +5,11 @@ gcloud auth configure-docker
 
 docker run --interactive --rm \
        --volume "$(pwd):/repo/" --workdir /repo/ \
-       tianqixu/chainerio \
-       sh -ex << EOD
-pip3 install --user -e .
-py.test tests -s -v
-flake8 chainerio
-flake8 tests
-autopep8 -r chainerio tests --diff | tee check_autopep8
-test ! -s check_autopep8
+       chainer/chainerio:latest \
+       bash << EOD
+source /root/.bashrc
+pyenv global 3.5.7 3.6.8 3.7.2
+tox
+pip install sphinx
```

Comment:
Why don't you make sphinx test and documentation dependencies as new `extras_require` entry in `setup.py` and run as additional deps in tox.ini ?